sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/ScannerForMSBuildInstaller.java

```diff
@@ -29,6 +31,7 @@
 import java.net.URL;
 import java.nio.file.Files;
 import java.nio.file.attribute.PosixFilePermissions;
+import javax.annotation.Nonnull;
```

Comment:
You can remove all the `@Nonnull` annotations as we use `@ParametersAreNonnullByDefault` in the _package-info.java_